src/std_api.rs

```diff
@@ -151,5 +151,10 @@ impl Command {
         self.config.supplementary_gids = Some(ids);
         self
     }
+
+    pub fn call_setpgid(&mut self, call_setpgid: bool) -> &mut Command {
```

Comment:
This should be named `make_group_leader`. And it should be in `std_api` but in `linux` module. The `std_api` is a place where we duplicate the `std::process::Command` API.

Also good docstring should accompany description of the method. With the reason of why would you like to make process a group leader.